gcapi/check_version.py

```diff
@@ -0,0 +1,49 @@
+import warnings
+from importlib.metadata import version as get_version
+
+import httpx
+from packaging import version
+
+
+class UnsupportedVersionError(Exception):
+    pass
+
+
+def check_version(base_url):
+    package_name = "gcapi"
+    try:
```

Comment:
I don't think the big try except clause is necessary if we do everything on our own servers now.